ctk/common/src/main/kotlin/org/codice/compliance/utils/GlobalSession.kt

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.utils
+
+import io.kotlintest.Description
+import io.kotlintest.extensions.TestListener
+import io.restassured.filter.cookie.CookieFilter
+import io.restassured.specification.RequestSpecification
+import org.codice.compliance.utils.GlobalSession.globalCookieFilter
+
+object GlobalSession : TestListener {
```

Comment:
I just decided this morning that I want to add javadoc to this object. I want to explain what it does overall but also why we're using a RestAssured `CookieFilter` in addition to a raw map of cookies. I would ideally like to use just the `CookieFilter`, but their API has some drawbacks. i.e. I can't manually add any cookies to the `CookieFilter`'s internal cookie store. The main reason this is an issue is because the `globalCookieFilter` doesn't get the cookies from the final response of the pluggable portion, and I have to include those cookies somehow. See https://github.com/connexta/saml-conformance/pull/158/files#diff-50a580a7096de25664d0f1799ccb018bR92.